src/components/calcite-inline-editable/calcite-inline-editable.scss

```diff
@@ -1,3 +1,7 @@
+:host {
+  @apply block;
```

Comment:
@eriklharper is this correct? seems input is block.

Reply:
I would advocate `inline-block` for this too.

Reply:
If I change this to `inline-block` the screener tests will change due to width differences. I will leave as now. If you think it should be changed I think a different issue is needed.